src/Logentries.CloudConfigurationProvider/CloudConfigProvider.cs

```
@@ -0,0 +1,24 @@
+﻿using System;
+using Logentries.Core.Configuration;
+using Microsoft.WindowsAzure;
+
+namespace Logentries.CloudConfigurationProvider
+{
+    public class CloudConfigProvider : IProvideConfiguration
+    {
+        public string GetSetting(string key)
+        {
+            try
+            {
+                return CloudConfigurationManager.GetSetting(key);
+            }
+
+            //TODO: Is there a specific exception we can catch?
+            catch (Exception)
```

if the config value must be there we should not swallow the exception. If it doesn't have to be there we should return a reasonable default.